packages/design-picker/src/types.ts

```diff
@@ -114,20 +101,9 @@ export interface Design {
 	preview_data?: PreviewData;
 	screenshot?: string;
 
-	/** @deprecated used for Gutenboarding (/new flow) */
+	/** @deprecated TODO: replace both with just stylesheet */
```

Comment:
I will resolve this in a follow-up PR (combining recipe, stylesheet, and theme) when I have another 20% time 🥲